content/commands/bgrewriteaof/index.md

```diff
@@ -50,7 +48,4 @@ Specifically:
 Since Redis 2.4 the AOF rewrite is automatically triggered by Redis, however the
 `BGREWRITEAOF` command can be used to trigger a rewrite at any time.
 
-Please refer to the [persistence documentation][tp] for detailed information.
-
-[tp]: /operate/oss_and_stack/management/persistence
-
+Please refer to the [persistence documentation]({{< relref "/operate/oss_and_stack/management/persistence" >}}) for detailed information.
```

Comment:
I think we avoid using "please" like this?

Reply:
Indeed. I wasn't focusing on copy-editing for this PR, but it's a solid point. Fixed in a subsequent commit.